steganography.py

```diff
@@ -128,6 +142,127 @@ def unmerge(img):
 
         return new_image
 
+    @staticmethod
+    def insert_hex(img, hex_code, x, y, rgb):
+        """Insert a hexadecimal pattern into the RGB values of an Image's pixels.
+
+        :param img: PIL Image object
+        :param hex_code: String of HEX code to be inserted into image
+        :param x: X Co-ordinate of pixel pattern insertion should begin with
+        :param y: Y Co-ordinate of pixel pattern insertion should begin with
+        :param rgb: 2D array specifying which RGB values should be overwritten
+        """
+        # Converts NumPy array of 1 & 0 values to True & False values respectively as it plays better with np.sum
+        rgb = np.array(json.loads(rgb), dtype=np.bool)
+
+        # Gets indices of each RGB binary value to be overwritten
+        rgb_indices = Steganography.__get_rgb_indices(rgb)
+
+        # Gets binary of Hex Code
+        bin_code = Steganography.__hex_to_bin(hex_code)
+
+        # Tuple used to store the image original size
+        original_size = img.size
+
+        # Check if binary code can be inserted
+        image_insert_size = ((original_size[0] - x) * (original_size[1] - y)) * len(rgb_indices)
```

Comment:
Could you please keep code in 80 columns (following [PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) recommendation)?